index.html

```diff
@@ -787,28 +790,42 @@
         <h2>
           <dfn>shippingAddress</dfn> attribute
         </h2>
-        <p>
-          <a>shippingAddress</a> is populated when the user provides a shipping
-          address. It is null by default. When a user provides a shipping
-          address, the <a>shipping address changed algorithm</a> runs.
+        <p class="note">
```

Comment:
> Hmm. I am less sure about the attributes than about the methods.

Agree. Removed "note".  Making statements of fact with non-RFC2119 words seems good (even if they mostly benefit developers). 

> Oh, also note that PaymentResponse currently uses a completely different style for its attributes, instead of having a heading per attribute :-/.

We should switch those to be headings (separate bug) - personally, I find it frustrating when attributes are not in the ToC - one should be able to see/access the API shape there too.